src/util/codemirror-utils.js

```diff
@@ -9,13 +9,28 @@ function highlightNode(codeMirror, node, filters, parentColor) {
   let color = parentColor;
 
   // If we aren't ignoring this token...
-  if (ignoredRules.indexOf(node.type) === -1  && rules[node.type]) {
-    color = rules[node.type].color; // Get the color for this token's type
+// <<<<<<< HEAD
```

Comment:
How about just combining the two into one filter, instead of having 2 separate filters for True and False?